### ISSUE TITLE
One-tab panel

### ISSUE THREAD

**Issue by unera** [mbnuqw/sidebery#1583]
### Description

It would be nice to have a panel for only one tab.

![изображение](https://github.com/mbnuqw/sidebery/assets/538980/b3b716ef-8c98-4c2f-8f2a-2acf7175ceda)

Such panels should:

- hide list of tabs
- but show panel icons/buttons
- show pined icons/buttons

So at screenshot sidebery panel could have width about 40px instead 200px.

The feature could be useful for messengers (for example).

**Comment by mbnuqw** [comment]
Blocked by https://bugzilla.mozilla.org/show_bug.cgi?id=1408077
